ejb3/src/main/java/org/jboss/as/ejb3/component/stateful/SerializableEJBProxy.java

```diff
@@ -22,39 +22,35 @@
 
 package org.jboss.as.ejb3.component.stateful;
 
-import org.jboss.ejb.client.EJBClient;
-import org.jboss.ejb.client.EJBClientContextIdentifier;
-import org.jboss.ejb.client.EJBLocator;
-
 import java.io.ObjectStreamException;
 import java.io.Serializable;
 
+import org.jboss.ejb.client.EJBClient;
+import org.jboss.ejb.client.EJBLocator;
+
 /**
- * A serializable EJB proxy which serializes the {@link EJBLocator} and the {@link EJBClientContextIdentifier}
- * associated with the EJB proxy
+ * A serializable EJB proxy which serializes the {@link EJBLocator} and the associated with the EJB proxy
  *
  * @author Jaikiran Pai
  */
-class SerializableEJBProxyWithEJBClientContextIdentifier implements Serializable {
+class SerializableEJBProxy implements Serializable {
```

Comment:
This class is (theoretically) no longer needed at all, since the "real" proxy should be all that is needed now.